test.wf

```diff
@@ -0,0 +1 @@
9945],[-0.02718320699720303,0.07191807737672182],[0.031788616341726986,0.1313028004811492],[-0.06604663885807543,0.1619326475364782],[-0.08906776692907928,-0.07052811429903999]],[[0.18347297917680513,0.013519137755730545],[-0.13490199091294217,-0.1503525364628285],[0.03425501856861925,0.07941431683286283],[0.0822115841419851,-0.05645011313764077],[0.001598124079984763,-0.06520423951606626],[-0.2361328592523463,0.0077172910971252985],[0.13715672845036977,0.0010220996041825246],[0.13154800419070867,-0.09571776964411041]],[[-0.10360917675526068,-0.07108303380375619],[0.037018592661958505,-0.02228238238038656],[0.07697994795898315,0.006270954028467296],[0.03612413830993247,0.03795890046604213],[0.21868806012143344,-0.00505513767481976],[-0.006997637274458658,0.09578450182526056],[-0.030096144363039427,0.09950907544377709],[-0.11989176712290767,-0.10113980313973883]],[[0.05143485875261264,0.028149053329254062],[-0.09363325118538408,-0.17233510810509944],[0.1796889595524001,0.15811594480490704],[0.08443359360884342,0.01662899436232297],[-0.12307883510517871,0.007426717638796132],[0.0037392764860829913,0.03946444487655621],[0.040483374734071875,0.0738571014086252],[-0.055687347801115616,-0.23032970821043694]],[[-0.025674265125667964,0.07182236140921619],[-0.06392751908920745,-0.10138184854692087],[-0.11209931478244174,0.10061526967835498],[-0.0533616219148198,0.07649586088099102],[0.026354482671456517,0.08312323206897546],[0.019071680784343972,0.05463868041788541],[-0.023313004297006452,0.12067119652126128],[-0.0825108801042168,-0.0932526632999322]],[[0.015872570716960414,0.031683873857233806],[0.08413459265830148,-0.009461348512865508],[0.06358240390877734,0.15078008726991893],[0.03625459164195814,0.1805249119020489],[-0.1399467566875341,-0.05268783339082248],[0.023452741555959507,-0.012802630804522246],[0.013013015979425899,0.030707930677882495],[-0.21619269062685298,-0.22958184041391497]],[[0.10016362791383161,0.19412447459330054],[-0.08239266304498638,-0.1075879987736796],[-0.18555146391857844,0.012233028502988703],[0.035359410228171725,0.05546861333987669],[-0.20395161103894766,0.16095135112359665],[0.050812528581159574,0.0805358439420753],[-0.029568672190314102,0.14897051705804296],[-0.05626444620051549,-0.029818632316574133]]],"a":[[-0.013452249878868762,0.0017094467054016467],[0.015122788156159837,-0.0028809888569498424],[0.009260123658455395,0.004228504547590641],[0.019291553258619087,0.0014229581437197802],[0.00920903696715213,-0.0022002138993473773],[0.008094743557323203,0.014820307816668585],[-0.006193199460089557,0.002262967682459739],[-0.02865863566408324,0.004495220350308481]],"b":[[-0.005300208289761835,0.008690582769794637],[0.0006143260107627523,0.008326950172987728],[-0.011379427302224564,-0.01124955498631998],[-0.0005445085316624998,0.014124774408673671],[-0.010261542708098821,0.0012491948236608979],[-0.0025952676042434583,-0.0005302460006722733],[0.001654736383255283,-0.0087652020874654],[-0.00084645287949665,0.000765011403867221]]}
```

Comment:
Do we need this file? Looks like a leftover from version 1.

Reply:
Yes this should be removed